roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -34,6 +34,33 @@
     mode: 0755
   changed_when: False
 
+- set_fact:
+    pvc_prefix: "{{ openshift_logging_es_pvc_prefix }}"
+
+- set_fact:
+    openshift_logging_pvc_name: "{{ openshift_logging_elasticsearch_pvc_name }}"
```

Comment:
I'm not sure I'm following the concern. Are there other places where we use `openshift_logging_pvc_name` that could be overwritten?